Hint on how to type the ÷ sign

ro-ble opened the pull request:
The ÷ sign is the only operator in this chapter which has no dedicated key on the keyboard. With the Unicode input being at the very end of the documentation, newcomers probably do not know how to write the integer division sign. Therefore, I added this short hint on how to type it. 

Best regards

ro-ble committed:
Hint on how to type the ÷ sign

The ÷ sign is the only operator in this chapter which has no dedicated key on the keyboard. With the Unicode input being at the very end of the documentation, newcomers probably do not know how to write the integer division sign. Therefore, I added this short hint on how to type it. 

Best regards

StefanKarpinski commented on doc/src/manual/mathematical-operations.md:
```diff
@@ -9,18 +9,18 @@ collection of standard mathematical functions.
+| `x - y`    | binary minus   | performs subtraction                                     |
+| `x * y`    | times          | performs multiplication                                  |
+| `x / y`    | divide         | performs division                                        |
+| `x ÷ y`    | integer divide | x / y, truncated to an integer, ÷ is typed by `\div<tab>`|
```
Rather than trying to squeeze it into the table, how about putting this note after the table?

ro-ble replied:
That makes sense. I'll write a sentence below the table. Another option could be to write a footnote with `[^1]`

ro-ble committed:
Move hint below the table

The hint was moved below the table. Additionally, a link was added to the manual chapter of Unicode Input.

ro-ble committed:
Added id to the Unicode chapter

staticfloat commented:
This PR seems to have broken the build; [example breakage](https://build.julialang.org/#/builders/20/builds/6988):

```
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: skipped.
[ Info: ExpandTemplates: expanding markdown templates.
[ Info: CrossReferences: building cross-references.
��� Warning: reference for 'Unicode-Input' could not be found in src/stdlib/REPL.md.
��� @ Documenter.CrossReferences /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/6vUwN/src/CrossReferences.jl:104
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
ERROR: LoadError: `makedocs` encountered an error. Terminating build
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] runner(#unused#::Type{Documenter.Builder.RenderDocument}, doc::Documenter.Documents.Document)
   @ Documenter.Builder /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/6vUwN/src/Builder.jl:255
 [3] dispatch(#unused#::Type{Documenter.Builder.DocumentPipeline}, x::Documenter.Documents.Document)
   @ Documenter.Utilities.Selectors /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/6vUwN/src/Utilities/Selectors.jl:170
 [4] #2
   @ /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/6vUwN/src/Documenter.jl:247 [inlined]
 [5] cd(f::Documenter.var"#2#3"{Documenter.Documents.Document}, dir::String)
   @ Base.Filesystem ./file.jl:106
 [6] #makedocs#1
   @ /buildworker/worker/package_linux64/build/doc/deps/packages/Documenter/6vUwN/src/Documenter.jl:246 [inlined]
 [7] top-level scope
   @ /buildworker/worker/package_linux64/build/doc/make.jl:216
in expression starting at /buildworker/worker/package_linux64/build/doc/make.jl:216
make[3]: *** [html] Error 1
make[2]: *** [docs] Error 2
```

Keno commented:
This was reverted. Sorry about that. Please feel free to open a new PR with the same change and we can figure out what happened to the build.

ro-ble commented:
I am sorry for messing with the build.. I have now opened a new pr #38762 and linked it to this one
